libraries/Wire/examples/WireMaster/WireMaster.ino

```diff
@@ -7,16 +7,16 @@ uint32_t i = 0;
 void setup() {
   Serial.begin(115200);
   Serial.setDebugOutput(true);
-  Wire.begin();
+  Wire1.begin();
 }
 
 void loop() {
   delay(5000);
 
   //Write message to the slave
-  Wire.beginTransmission(I2C_DEV_ADDR);
-  Wire.printf("Hello World! %u", i++);
-  uint8_t error = Wire.endTransmission(true);
+  Wire1.beginTransmission(I2C_DEV_ADDR);
+  Wire1.printf("Hello World! %u", i++);
+  uint8_t error = Wire1.endTransmission(true);
```

Comment:
@SuGlider Do we need to change the example to Wire1? Not all boards will have Wire1 pins defined. :)
And the example won't work correctly now, when `Wire1.begin()` called, but requesting bytes from **Wire** `Wire.requestFrom(I2C_DEV_ADDR, 16);`. Take a look :)

Reply:
Actually, it was commited by mistake... I'll revert it.